morpheus-graphql-subscriptions/src/Data/Morpheus/Subscriptions/Apollo.hs

```diff
@@ -84,7 +87,10 @@ instance FromJSON a => FromJSON (ApolloSubscription a) where
   parseJSON = withObject "ApolloSubscription" objectParser
     where
       objectParser o =
-        ApolloSubscription <$> o .:? "id" <*> o .: "type" <*> o .:? "payload"
+        ApolloSubscription
+          <$> o .:? "id"
+          <*> o .: "type"
```

Comment:
since we have now  ApolloResponseType, would be great if use it directly at  line 78  `apolloType :: ApolloResponseType` ;)

Reply:
WIP with last commit - will resume tomorrow. Will need to update various client code as well, as the DS is used not just for responses, but requests as well.

Reply:
Done!